hydra-tui/hydra-tui.cabal

```diff
@@ -82,12 +79,18 @@ library
     , async
     , base
     , brick
+    , cardano-ledger-core
```

Comment:
Suggestion: It seems odd to have the TUI directly dependent on cardano's stuff. Would it not make sense to use the `Hydra.Ledger.Cardano` API instead?